editor/scene_tree_dock.cpp

```diff
@@ -3481,6 +3487,88 @@ void SceneTreeDock::_create_remap_for_resource(Ref<Resource> p_resource, HashMap
 	}
 }
 
+void SceneTreeDock::_list_all_subresources(PopupMenu *p_menu) {
+	p_menu->clear();
+
+	List<Pair<Ref<Resource>, Node *>> all_resources;
+	if (edited_scene) {
+		_gather_resources(edited_scene, all_resources);
+	}
+
+	HashMap<String, List<Pair<Ref<Resource>, Node *>>> resources_by_type;
+	HashMap<Ref<Resource>, int> unique_resources;
+
+	for (const Pair<Ref<Resource>, Node *> &pair : all_resources) {
+		if (!unique_resources.has(pair.first)) {
+			resources_by_type[pair.first->get_class()].push_back(pair);
+		}
+		unique_resources[pair.first]++;
+	}
+
+	for (KeyValue<String, List<Pair<Ref<Resource>, Node *>>> kv : resources_by_type) {
+		p_menu->add_icon_item(EditorNode::get_singleton()->get_class_icon(kv.key), kv.key);
+		p_menu->set_item_as_separator(-1, true);
+
+		for (const Pair<Ref<Resource>, Node *> &pair : kv.value) {
+			String display_text;
+			if (pair.first->get_name().is_empty()) {
+				display_text = vformat(TTR("<Unnamed> at %s"), pair.second->get_name());
+			} else {
+				display_text = pair.first->get_name();
+			}
+
+			if (unique_resources[pair.first] > 1) {
+				display_text += " " + vformat(TTR("(used %d time(s))"), unique_resources[pair.first]);
```

Comment:
At least in English this will also be plural since it's more than 1, so it can be `used %d times`.

For localization we might want to actually use `TTRN` though, but we don't use it much currently, that's work for a big refactor pass (notably for Polish I think which has more complex plural forms?).